apps/desktop/tests/ci/e2e-webview2-env.test.js

```
@@ -0,0 +1,54 @@
+    });
+
+    it('injects default browser arguments and optional user data folder', () => {
+        const env = { TOUCHAI_E2E: '1' };
+
+        expect(withE2eWebView2Env(env)).toEqual({
+            TOUCHAI_E2E: '1',
+            WEBVIEW2_ADDITIONAL_BROWSER_ARGUMENTS:
+                DEFAULT_E2E_WEBVIEW2_ADDITIONAL_BROWSER_ARGUMENTS,
+        });
+
+        expect(withE2eWebView2Env(env, { userDataFolder: 'D:\e2e-user-data' })).toEqual({
```

ESLint

[error] Unnecessary escape character: \e.

(no-useless-escape)